### PR TITLE
Proposing changes to terminology and lens FOV as per #3255

### PR DESCRIPTION
**Pull request by sandyol55**
First git commit so apologies in advance for any errors.

### CODE REVIEW

**Comment by aallan** [comment]
Closes #3255

**Comment by davidplowman** [comment]
The whole field-of-view vs. angle-of-view vs. field-angle vs. angular-field-of-view all seems rather confusing! I suppose angle-of-view or field-angle is probably better for angular measurements because field-of-view is sometimes regarded as being a distance, not an angle?

Anyway, changing the terminology seems fine to me. I notice the actual numbers have changed too - do I understand that someone has obtained or measured more accurate numbers?

File: documentation/asciidoc/accessories/camera/lens.adoc [hunk 1]
```diff
@@ -16,12 +16,9 @@ We recommend two lenses, a 6mm wide angle lens and a 16mm telephoto lens. These
 2+| Image format | 1" | 1/2"
 2+| Aperture | F1.4 to 1.6 | F1.2
```
Comment (nathan-contino) [inline documentation/asciidoc/accessories/camera/lens.adoc:17]:
```suggestion
2+| Aperture | F1.4 to F16 | F1.2
```
Suggestion: update this row based on the recommendations in https://github.com/raspberrypi/documentation/issues/3255

**Review by nathan-contino — APPROVED** [review]
LGTM as long as we pull in my suggestion. I'll merge that right now, this PR should be good to merge any time after that.

**Comment by nathan-contino** [comment]
Good to merge.

**Review by aallan — APPROVED** [review]

**Pull request merged**